support/build/README.md

```diff
@@ -169,7 +169,7 @@ The following environment variables are highly recommended (see section *Underst
 
 The following environment variables are optional:
 
-- `S3_REGION`, to be set to the AWS region name (e.g. "`s3-eu-west-1`") for any non-standard region, otherwise "`s3`" (for region "us-east-1")
+- `S3_REGION`, to be set to the AWS region name (e.g. "`s3.eu-west-1`") for any non-standard region, otherwise it will default to "`s3.us-east-1`".
```

Comment:
The hyphenated form is also considered legacy:
https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html#s3-legacy-endpoints